### DISCUSSION TITLE
Ctrl+Click to jump to declaration in IDE

### DISCUSSION THREAD

**Discussion by YogiYang** [twinbasic/twinbasic#475] [Ideas]
Hello,

As I am using Delphi a lot in recent days I have come to appreciate a feature in its IDE. Which allows me to navigate to a function/procedure/variable declaration by Ctrl + Click. This also works across modules. Meaning if I click on a function/procedure/variable and if that resides in another module the IDE should open that module and take the user to the declaration.

@WaynePhillipsEA can you implement something like this in the IDE?

**Comment by WaynePhillipsEA** [comment]
This should already be working, as Ctrl+Click is mapped to 'Go to definition' (F12).
